<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>WelcomeWindow</class>
 <widget class="QMainWindow" name="WelcomeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>461</width>
      <height>191</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Academy Engraved LET</family>
      <pointsize>82</pointsize>
     </font>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="text">
     <string>RogueLike</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignCenter</set>
    </property>
   </widget>
   <widget class="QPushButton" name="startGameButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>320</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Start Game</string>
    </property>
   </widget>
   <widget class="QPushButton" name="storeButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>420</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Store</string>
    </property>
   </widget>
   <widget class="QPushButton" name="continueGameButton">
    <property name="geometry">
     <rect>
      <x>350</x>
      <y>370</y>
      <width>100</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>Resume Game</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>330</y>
      <width>58</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>With:</string>
    </property>
   </widget>
   <widget class="QComboBox" name="playerChooser">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>320</y>
      <width>103</width>
      <height>32</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
